lr_face/models.py

```diff
@@ -54,5 +55,96 @@ def score_for_pair(self, pair):
         img2 = resize_and_normalize(pair[1], self.model.input_shape[1:3])
         img1_representation = self.model.predict(img1)[0, :]
         img2_representation = self.model.predict(img2)[0, :]
-        score = spatial.distance.cosine(img1_representation, img2_representation)
+        score = spatial.distance.cosine(img1_representation,
+                                        img2_representation)
         return score
+
+
+class FinetuneModel(tf.keras.Model):
```

Comment:
dit is feitelijk een schil om je embedding model heen om m 3x aan te kunnen roepen. Hoewel ik snap dat dit een keras Model is denk ik dat een naam als 'TripletEmbedder' of iets in die trend beschrijvender is.

Reply:
Goed idee! Dan wordt uit de naam ook duidelijk dat het specifiek voor triplets werkt en eigenlijk niet voor andere manieren van finetunen.